order/buyer/issue/delivery-schedule.md

```diff
@@ -1,31 +1,30 @@
 ---
 description: >-
-  Choose between the native or simple delivery schedule
+  Choose between delivery schedule or single delivery per order line
```

Comment:
Might be handy to make this page a 1 on 1 copy of https://app.gitbook.com/o/-LNyIUZSvpZWZ81yEr7V/s/-LNyJ9UuwLCjTvA2sqQR-887967055/~/revisions/2b1C2rybUsksVl4uYRuy/introduction/api/delivery-schedule

Reply:
You are right, there is a great overlap, the general one, is general, and this one is focused on issuing.
The delivery schedule vs single delivery be rewritten as part as the new API guide.